src/main/java/sg/ncl/testbed_interface/ResumableInfo.java

```diff
@@ -0,0 +1,35 @@
+package sg.ncl.testbed_interface;
+
+/**
+ * Created by dcsjnh on 11/29/2016.
+ *
+ * References:
+ * [1] https://github.com/23/resumable.js/blob/master/samples/java/src/main/java/resumable/js/upload/ResumableInfo.java
+ */
+public class ResumableInfo {
+
+    public int      resumableChunkSize;
+    public long     resumableTotalSize;
+    public String   resumableIdentifier;
+    public String   resumableFilename;
+    public String   resumableRelativePath;
+
+    public static class ResumableChunkNumber {
+        public ResumableChunkNumber(int number) {
+            this.number = number;
+        }
+
+        public int number;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1213)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make number a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)